packages/worker/src/worker.ts

```diff
@@ -165,7 +165,7 @@ export interface WorkerOptions {
   /**
    * Maximum number of Activities to execute concurrently.
    * Adjust this to improve Worker resource consumption.
-   * @default 200
+   * @default 10
```

Comment:
:sunglasses:

Reply:
wait why is this 😎 i thought it would be 😢

Reply:
It's a sardonic :sunglasses:, lol

Reply:
It bumped it up to 100, I got it wrong, it's 10 WFTs and 100 ATs.
After we merge the bundler branch I'll bump it back up to 100.